core/templates/dev/head/editor/ExplorationHistory.js

```diff
@@ -53,6 +54,32 @@ oppia.controller('ExplorationHistory', [
     }
   });
 
+  $scope.submitVersion = function($event, versionNumber) {
+    var checkbox = $event.target;
+    var id = $scope.versionArr.indexOf(versionNumber);
```

Comment:
Suggest renaming versionArr to something more descriptive. It's not very clear what this variable is for. Perhaps something like "selectedVersionsArray".

Also it will be good add a comment that this should contain a maximum of two elements.